nuxeo-drive-client/nxdrive/tests/test_watchers.py

```diff
@@ -228,6 +230,23 @@ def test_local_scan_encoding(self):
         self.assertFalse(remote.exists(u'/Accentue\u0301.odt'))
         self.assertFalse(remote.exists(u'/P\xf4le applicatif/e\u0302tre ou ne pas \xeatre.odt'))
 
+    def test_watchdog_space_remover(self):
+        if AbstractOSIntegration.is_windows():
+            raise SkipTest("Windows cannot have file ending with a space")
+        local = self.local_client_1
+        remote = self.remote_document_client_1
+        # Start engine
+        self.engine_1.start()
+        # Wait for test workspace synchronization
+        self.wait_sync()
+        local.make_file(u'/', u'Accentue\u0301.odt ', u'Content')
+        self.wait_sync()
+        self.assertTrue(remote.exists(u'/Accentue\u0301.odt'))
```

Comment:
As we talk about it, here you can add:

    self.assertFalse(remote.exists(u'/Accentue\u0301.odt '))